packages/strapi-plugin-content-type-builder/admin/src/containers/Form/forms.json

```diff
@@ -259,6 +259,7 @@
             "value": "integer",
             "items": [
               { "name": "content-type-builder.form.attribute.item.number.type.integer", "value": "integer" },
+              { "name": "content-type-builder.form.attribute.item.number.type.biginteger", "value": "big integer" },
```

Comment:
I think you should change it to:
` { "name": "content-type-builder.form.attribute.item.number.type.biginteger", "value": "biginteger" },`